internal/controller/ingress/httpsedge_controller.go

```diff
@@ -1028,3 +1029,30 @@ func (r *HTTPSEdgeReconciler) takeOfflineWithoutAuth(ctx context.Context, route
 
 	return nil
 }
+
+func (u *edgeRouteModuleUpdater) setEdgeRoutePolicies(ctx context.Context, route *ngrok.HTTPSEdgeRoute, routeSpec *ingressv1alpha1.HTTPSEdgeRouteSpec) error {
+	log := ctrl.LoggerFrom(ctx)
+	policies := routeSpec.Policies
+	client := u.clientset.Policies()
+
+	module := policies.ToNgrok()
+
+	// Early return if nothing to be done
+	if module == nil {
```

Comment:
I think my only nit for the PR is the variable name `module` is a little confusing since its a `EndpointPolicies` struct. Reading it i was a little confused about if we are comparing modules to policies or what. I'm new to the policy stuff so it may just be a knowledge gap though

Reply:
The reason why I called this a `module` instead of an `EndpointPolicy` is to match with the pattern set-out in this file for other modules. Where the `ngrok-api-go` struct is just named a `module`.

Reply:
renamed for readability: https://github.com/ngrok/kubernetes-ingress-controller/pull/334/commits/41f50da03fb7e76562424e2f4db9793179a0f45c